### DISCUSSION TITLE
freie GPIOs steuern

### DISCUSSION THREAD

**Discussion by ESP-Raider** [softwarecrash/Victron2MQTT#22] [General]
Hi,
ist es möglich, über MQTT einen der freien GPIO zu steuern (EIN/AUS)?
Kann ich einen Befehl zum ESP publishen, um einen Ausgang zu steuern?

Viele Grüße

**Comment by softwarecrash** [comment]
aktuell nur den  steuerausgang von victron (D6) der  kann auch  anderweitig benutzt werden

**Reply by hoschiking** [reply]
Das wäre prima, wenn mehrere frei nutzbare GPIO geben würde, die man per MQTT schalten könnte.
